packages/react/src/components/AmplifyProvider/index.tsx

```diff
@@ -2,9 +2,10 @@ import { IdProvider } from '@radix-ui/react-id';
 import { ReactNode } from 'react';
 
 import { AmplifyContext } from './AmplifyContext';
-import { createTheme, defaultTheme, Theme } from '@aws-amplify/ui';
+import { createTheme, defaultTheme, Theme as UiTheme } from '@aws-amplify/ui';
 import * as primitives from '../../primitives';
 
+export type Theme = UiTheme;
```

Comment:
I could've sworn `export { Theme } from "@aws-amplify/ui"` would work, but that's a nit.

You can manually validate by checking the `dist/index.d.ts` output after `yarn build`, too.